static/scss/detail/learning-techniques.scss

```diff
@@ -0,0 +1,75 @@
+// sass-lint:disable mixins-before-declarations
+@import "common";
+
+.learning-techniques-block {
+  padding-top: 70px;
+  padding-bottom: 50px;
+
+  h1 {
+    text-align: center;
+    font-size: 40px;
+    line-height: 44px;
+    margin: 0 0 50px;
+    color: $primary;
+    font-weight: 600;
+    text-transform: uppercase;
+  }
+}
+
+.learning-techniques-list {
+  margin: 0;
+  overflow: hidden;
+  padding: 0;
+  list-style: none;
+  letter-spacing: -.32em;
+  position: relative;
+
+  @include media-breakpoint-up(md) {
+    display: flex;
+    flex-wrap: wrap;
```

Comment:
I'm pretty sure you can define these two styles for all screen widths. Looked the same to me when I tested it

Reply:
@gsidebo regarding you last comment below 
`While decreasing the browser width, once the items go to a single column, their alignment is off.`
After that change, it is effecting the scree. I suggest we should revert to back this rule here.

Reply:
OK, that will work